events/message-create.js

```diff
@@ -46,9 +48,7 @@ module.exports = {
      */
     let isAdminMessage = false;
     try {
-      isAdminMessage = message.member.roles.cache.some(
-        (r) => config.roles.adminRolesName.includes(r.name),
-      );
+      isAdminMessage = isAdmin(message.member);
     } catch (e) {
       //  The only 'con' is a command or message gets ignored.
     }
```

Comment:
Looks like this try/catch (and the one below for isMessageAuthorNobot) is only to ignore null errors with `message.member`.

Both of these try catches can be reduced to:

```js
    const isAdminMessage = isAdmin(message.member);
    const isMessageAuthorNobot = message.member?.roles.cache.has(
      config.roles.NOBOTRoleId,
    );
```
With `isAdmin` using `?.` for `return member?.roles.cache.....`